assets/css/iziModalToast/iziModal.css

```diff
@@ -244,6 +244,7 @@
     margin: 0 0 -6px 0;
     width: 100%;
     transition: height 0.3s ease;
+    height: 100% !important;

```

Comment:
Lets give it `95%`

Reply:
Ok i will do that and see how it looks like thanks @muarachmann